### PR TITLE
feat: assign taxonomy to organizations [TEMP]

### PR DESCRIPTION
**Pull request by rpenido**
TEMP

### CODE REVIEW

File: src/taxonomy/manage-orgs/ManageOrgsModal.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.manage-orgs {
+  overflow: visible !important;
+
+  .pgn__modal-body {
+    overflow: visible;
```
Comment (ChrisChV) [inline src/taxonomy/manage-orgs/ManageOrgsModal.scss:11]: @rpenido Could you add the reason of this style in a comment?

**Review by ChrisChV — COMMENTED** [review]
@rpenido I think you need to add some padding here. Like the modal on [Figma](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=9715-51696&mode=design&t=RQE5B5zPri3Ez4TU-4)

photo_2023-12-20_08-34-10

File: src/taxonomy/manage-orgs/ManageOrgsModal.scss (hunk 1 above)
Comment (rpenido) [reply src/taxonomy/manage-orgs/ManageOrgsModal.scss:11]: Done [9fbe76e](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/9fbe76ebefb4ec86d57b0fce3d171f5bb84cddaa)!

Also updated the spacing here: [aca5ce5](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/aca5ce5873580c50817e9404b27de54bfc71a744)

image

**Review by ChrisChV — APPROVED** [review]

**Comment by ChrisChV** [comment]
@rpenido Looks good, great work! :+1:
- [X] I tested this: I tested all the workflow.
- [X] I read through the code and considered the security, stability and performance implications of the changes.
- [X] I tested that the UI can be used with a keyboard only (tab order, keyboard controls).
- [X] Includes tests for bugfixes and/or features added.
- [X] Includes documentation

**Comment by rpenido** [comment]
Closed in favor of https://github.com/openedx/frontend-app-course-authoring/pull/760

**Pull request closed without merging**